packages/core/src/types.ts

```diff
@@ -6,7 +6,7 @@
 // Imports
 //------------------------------------------------------------------------------
 
-import { JSONSchema4 } from "json-schema";
+import type { JSONSchema4 } from "json-schema";
```

Comment:
`import type` is required for the types to compile when the TS option [`verbatimModuleSyntax`](https://www.typescriptlang.org/tsconfig/#verbatimModuleSyntax) is specified.